### ISSUE TITLE
Disabling chat history keeps the history button

### ISSUE THREAD

**Issue by michaelfester** [motifland/markprompt-js#206]
**Describe the bug**

When `chat.history.enabled = false`, the history button is still present in the UI.

<img width="1089" alt="CleanShot 2023-10-05 at 22 33 31@2x" src="https://github.com/motifland/markprompt-js/assets/504893/f516dedf-ffa3-44cc-8baf-01c674383a0e">

**Comment by nickrttn** [comment]
Yes, because even if chat history is disabled we keep history for a session, just not longer than that

**Comment by michaelfester** [comment]
Yes but we shouldn't display the button since it serves no purpose, right?
